### PR TITLE
Credits in the game

### LINKED ISSUE #2084: Make a script to retrieve the game credits from the wiki
and make into a json file that is included in the repo and game releases. This will be used then once we get a credits in the game for easily updating the credits in the game once we add them.

### PR DESCRIPTION
**Pull request by hhyyrylainen**
**Brief Description of What This PR Does**

adds a script to fetch the credits and a scene that displays the game credits

**Related Issues**

closes #2084

**Progress Checklist**

Note: before starting this checklist the issue should be marked as non-draft.

- [x] PR author has checked that this PR works as intended and doesn't
      break existing features:
      https://wiki.revolutionarygamesstudio.com/wiki/Testing_Checklist
      (this is important as to not waste the time of Thrive team
      members reviewing this PR)
- [ ] Initial code review passed
- [ ] Functionality is confirmed working by another person (see above checklist link)
- [ ] Final code review is passed and code conforms to the 
      [styleguide](https://github.com/Revolutionary-Games/Thrive/blob/master/doc/style_guide.md).

Before merging all CI jobs should finish on this PR without errors, if
there are automatically detected style issues they should be fixed by
the PR author.

### CODE REVIEW

File: scripts/retrieve_credits.rb [hunk 1]
```diff
@@ -0,0 +1,219 @@
...
+require_relative 'json_helpers'
+require_relative 'folder_detector'
+
+DEVELOPERS_PAGE = 'https://wiki.revolutionarygamesstudio.com/wiki/Team_Members'
+DONATIONS_PAGE = 'https://wiki.revolutionarygamesstudio.com/wiki/Donations'
+
+# +31 is here to guarantee donations made in the last day of a month will be included properly
+DONATION_DISPLAY_CUTOFF = (365 + 31) * 60 * 60 * 24
+
+FILE_DOWNLOAD_OLDNESS = 60 * 60 * 24
```
Comment (adQuid) [inline scripts/retrieve_credits.rb:20]: age?

Comment (hhyyrylainen) [reply scripts/retrieve_credits.rb:20]: Replacing "oldness" with "age" doesn't actually make the name descriptive. This is the threshold when to consider a file old. Now that I'm looking at this I'm wondering why I didn't add the word "threshold" in there.

Comment (hhyyrylainen) [reply scripts/retrieve_credits.rb]: Made a commit where I renamed it, and it now includes the word "age" in it.

**Review by Kasterisk — COMMENTED** [review]
Would be nice to add the return to main menu on escape functionality to the credits menu.

**Comment by hhyyrylainen** [comment]
I've spent 2 workdays on this already, so I plan on opening 3 new issues after this is merged:
- issue for making translateable the team names and other dynamic sections in the credits
- fast forward and skip (need to hold for 3 seconds) buttons for the credits
- implement two columns for use in a few of the longer lists of names in translation

**Pull request merged**